app/models/user_access.rb

```diff
@@ -75,6 +91,10 @@ def accessible_district_regions(action)
     Region.district_regions.where(source_id: facility_group_ids)
   end
 
+  # We determine accessible blocks via the parent district authorization - so we need to jump through some hoops
+  # here given our current user access model.
```

Comment:
Shall we put the actual auth rule in this comment while we're here? Eg.

A block is accessible to a user only if the parent district is also accessible.